README.md

```diff
@@ -33,8 +29,7 @@ The samples were developed and tested using [Mars 4.5.0](https://eclipse.org/mar
 
 ## Using remote input PDF files with the samples
 
-By default, the samples run with pre-packaged input PDF files that are treated as resources of the samples. However, you can use your own input PDF files 
-including those that are stored remotely on a shared network with the samples. As long as the shared network where the PDF files reside is mounted on your local machine it's treated a part of your local file system.
+By default, the samples run with pre-packaged input PDF files that are treated as resources of the samples. However, you can use your own input PDF files, including those that are stored remotely on a shared network with the samples. As long as the shared network where the PDF files reside is mounted on your local machine it's treated a part of your local file system.
```

Comment:
Did any content change here or is it just putting things on one line?

We try to avoid putting everything on one line so that lines do not run on forever (there should be a limit to the number of characters per line, when writing in markdown it doesn't matter if the content is broken across multiple lines it stitches them together when rendered).

We try to avoid these changes because they end up as noise in the pull request and it is difficult to see what changed (if anything other than being on multiple lines).